packages.yml

```diff
@@ -1,5 +1,5 @@
 packages:
   - package: dbt-labs/dbt_utils
-    version: 0.8.6
+    version: [">=0.8.6", "<2.0.0"]
   - package: dbt-labs/spark_utils
```

Comment:
@b-per -- actually pretty sure we can remove the dependency on spark utils in this PR -- does that sound right to you?

Reply:
looks like we were just using this for the spark shims, which we have now moved into this package until dbt 1.3 is released! we are safe to remove it